packages/atomic/src/components.d.ts

```diff
@@ -3499,6 +3525,18 @@ declare namespace LocalJSX {
          */
         "store"?: AtomicInsightStore;
     }
+    interface AtomicInsightResultAction {
+        /**
+          * Specify the result action icon to display.
+         */
+        "icon"?: string;
+        /**
+          * The text tooltip to show on the result action icon
+         */
+        "label"?: string;
+    }
```

Comment:
:suspect: why does those interface keys are wrapped in double quotes?

Reply:
they are all wrapped in double quotes 🤔

Reply:
Indeed they are as this is the generated components file 😅